src/utils.js

```diff
@@ -6,7 +6,32 @@ function remove(array, element) {
   array.splice(array.indexOf(element), 1);
 }
 
+const mockGithub = {
```

Comment:
So you can make a separate `github-mock-spec.js` file in the spec directory instead of the utils file